### PR TITLE
Fix Minio Browser Login Missing Information

### PR DESCRIPTION
**Pull request by PatrickXYS**
*Issue #1 , if available:*

*Description of changes:*
Add information about Minio Browser Login information.

@chanwit @dlespiau

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ Then install the COVID ML Profile, which is a portable profile containing all FK
 
 After the profile installed, wait until all PODs ready and FK-Covid can be accessed via URLs retrieved from `kubectl get svc -n kubeflow`.
 
+After accessed to Minio browser and login page asks you to fill in `Access Key` and `Secret Key`, they're hardcoded as `minio` and `minio123`.
```
Comment (chanwit) [inline README.md]: Could you please rephrase it to this?

When opening browser to the MinIO page, and the login page will ask you to fill in `Access Key` and `Secret Key`. Please use the default keys as `minio` and `minio123`.

**Review by chanwit — CHANGES_REQUESTED** [review]

**Comment by chanwit** [comment]
@PatrickXYS Thank you for the first ever PR on this repo. I'm very happy and really I really appreciate.

There's small things to fix, like the name Minio => MinIO.

File: README.md (hunk 1 above)
Comment (PatrickXYS) [reply README.md]: Sure, no problem, let me rephrase it.

**Comment by PatrickXYS** [comment]
@chanwit I'd love to contribute more if there's an issue on the EKS side or anything I could help with. This is a great product, look forward to today's live!

**Review by chanwit — APPROVED** [review]

**Comment by chanwit** [comment]
Cheers!
thank you @PatrickXYS and congrats for the first PR being merged!!
